### ISSUE TITLE
[Spotify Player] Menu Bar Player not working

### ISSUE THREAD

**Issue by gemmaayt** [raycast/extensions#11845]
### Extension

https://www.raycast.com/mattisssa/spotify-player

### Description

Hello! I would like to ask for help as I would love to have the menu bar player but it doesn't seem to be working for me but in the settings nothing seems to be wrong. The menu bar player doesn't show up for me and I've tried redownloading the extension and relogging into spotify multiple times. Please let me know if anything could be done for this! Thank you!

### Steps To Reproduce

![Screenshot 2024-04-17 at 7 21 34 pm](https://github.com/raycast/extensions/assets/167300338/58debd90-d65d-400b-a10e-07f61feb0488)

### Current Behaviour

It doesn't show up on the menu bar
![Screenshot 2024-04-17 at 7 22 20 pm](https://github.com/raycast/extensions/assets/167300338/87570bb3-84df-4559-8197-a46d735bb115)

### Expected Behaviour

It should show up on the menu bar

**Comment by thomaslombart** [comment]
It seems unchecking "Hide icon when idle" does the trick. Something related to that doesn't seem to be right. It also appears that this is a duplicate of issue https://github.com/raycast/extensions/issues/10466, so I'll mark it as such.

**Comment by gemmaayt** [comment]
Hi, I've looked at issue #10466 but it's not that mine isn't refreshing but it has an error? I've uploaded a screenshot below as shown, i've tried multiple times of relogging into spotify, deleting the extension and redownloading it and quitting raycast and reinstalling as well and nothing has worked. I'd appreciate any help for this, thank you!
<img width="285" alt="Screenshot 2024-05-24 at 4 50 20 pm" src="https://github.com/raycast/extensions/assets/167300338/94b70c51-bbd8-4e60-a70a-c2181f934851">
